modules/microshift-updating-container-image-new-app.adoc

```diff
@@ -0,0 +1,94 @@
+// Module included in the following assemblies:
+//
+// * unused* was proposed for the embedding offline apps assemblies
+
+:_content-type: PROCEDURE
+[id="microshift-updating-container-image-new-app_{context}"]
+= Updating an existing container image for a new application
+
+After you have embedded workloads or applications, update the existing {op-system-ostree-first} image.
```

Comment:
I would make this section more generic without including concrete commands and their outputs. 
Currently, it's a bit misleading because we describe the end result without showing in details how it starts.

Reply:
I think this is currently unused, so we can simply remove it. Also, updating an app is the same as updating microshift, so this will all be covered by the update docs we have. lets delete it here.

Reply:
Yeah, it is not being used. I put the module in the Unused topics folder in the repo.